components/footer.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from 'next/link'
+
+const Footer = () => {
+  return (
+    <Link href="../">
```

Comment:
`next` links like to have an anchor tag inside of them. Kinda like this:

```
<Link href="/">
  <a>Back to main</a>
<Link>
```

Reply:
Of note is that you don't need to have any attributes on the anchor tag. In fact, I think Next will scream at you if you do.